app/models/notification.rb

```diff
@@ -10,9 +26,9 @@ class Notification < ActiveRecord::Base
   delegate :eventable, :to => :event
 
   scope :unviewed, -> { where(viewed: false) }
+  scope :beta_notifications, -> { joins(:event).where(:events => {:kind => BETA_NOTIFICATIONS}) }
```

Comment:
Brilliant! I think we want to stay away from the hash rocket though (as a matter of style)

```
joins(:event).where(events: { kind: BETA_NOTIFICATIONS })
```